components/CardsWrapper.jsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
```

Comment:
```suggestion
import styled from 'styled-components/macro';
```
Das macht das Debuggen leichter, da er dann den Komponentennamen mit in den Klassennamen integriert.

Reply:
das ist nicht so einfach in next und würde ich für jetzt rauslassen